lib/pages/home/home_page.dart

```diff
@@ -105,10 +118,11 @@ class HomePage extends StatelessWidget {
   Future<void> _callFetchPlan(BuildContext context) async {
     final homePageCubit = context.read<HomePageCubit>();
     final appReviewCubit = context.read<AppReviewCubit>();
-    final settingPanelCubit = context.read<SettingPanelCubit>();
+    final payloadDataPlanCubit = context.read<PayloadDataPlanCubit>();
     final correlationId = context.read<PreferencesCubit>().state.correlationId;
+    await homePageCubit.refreshCurrentRoute();
```

Comment:
@GustavoTCh  what is the objective of this line? is generating a loading loop issue, after select a place